synapse/crypto/keyring.py

```diff
@@ -815,87 +814,67 @@ async def _fetch_keys(
 
         async def get_key(key_to_fetch_item: _FetchKeyRequest) -> None:
```

Comment:
Similarly should this be `get_keys` now?

Reply:
```suggestion
        async def get_keys(key_to_fetch_item: _FetchKeyRequest) -> None:
```